styles.css

```diff
@@ -0,0 +1,59 @@
+*{
+  text-align: center;
+  box-sizing: border-box;
+}
+
+#tasks{
+  display:flex;
+  flex-direction: column;
+  align-items: stretch;
+  text-align: left;
+  background-color: blue;
+  height: 400px;
+  width: 600px;
+  margin: auto;
+  
+  overflow-y: scroll;
+}
+
+.task{
+  background-color: white;
+  text-align:left;
+  padding: 1rem;
+  border-radius: 10px;
+  margin: 0.5rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+#input-area{
+  background-color: lightblue;
+  height:100px;
+  width: 500px;
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: left;
+  margin:auto; 
+  
+  position:relative;
+  right:20px
+}
+
+#text-input{
+  text-align:left;
+  padding: 0.5rem;
+  margin:0.25rem;
+  border-radius: 10px;
+  outline: none !important;
+  border: none;
+}
+
+#clear-button{
+  background-color:yellow;
+  width:20rem;
+  position:relative;
+  left: 40px;
+  
+
+}
```

Comment:
Generally we would put a newline at the end of each file. This is best practice, albeit a bit pedantic.